### ISSUE TITLE
Attempt to avoid suggestions about adding comments

### ISSUE THREAD

**Issue by esauser** [Codium-ai/pr-agent#318]
A lot of the suggestions we get are to add comments.  These have not once been helpful.  Would you consider additional prompts to avoid suggestions for adding comments?

**Comment by mrT23** [comment]
have you tried using the "extra_instructions" field ?
that's exactly its purpose, to enable teams to easily and quickly customize and adjust the output

**Comment by esauser** [comment]
@mrT23 yes.  I opened this issue because I thought it might be worthwhile for everyone to have be a more permanent.

**Comment by mrT23** [comment]
see https://github.com/Codium-ai/pr-agent/pull/327
